app/views/devise/registrations/new.html.erb

```diff
@@ -26,6 +36,16 @@
     <%= f.password_field :password_confirmation, autocomplete: "off" %>
   </div>
 
+   <div class="field">
+    <%= f.label :profile_photo %><br />
+    <%= f.file_field :profile_photo, autofocus: true, autocomplete: "Profile Photo" %>
+  </div>
+
+   <div class="field">
+    <%= f.label :interests %><br />
+    <%= f.text_field :interests, autofocus: true, autocomplete: "Interests" %>
+  </div>
+
```

Comment:
When trying to create an account, the information is being saved to the database behind the scenes, but we receive the following error after Sign up is clicked. This issue also prevents us from logging in to try to update our account:

![screen shot 2018-08-20 at 8 41 38 pm](https://user-images.githubusercontent.com/36312228/44375603-1cde4080-a4ba-11e8-8352-def66338ec53.png)

Reply:
This appears to be an error related to Admin, not User

Reply:
Getting this to work alright now, think it was just a migrations issue.